Assets/Scripts/UI/PlayerHUDController.cs

```diff
@@ -36,6 +36,7 @@ public class PlayerHUDController : MonoBehaviour
     private Material damageBorder;
 
     private const float ammoSpinDegrees = 30;
+    private const float availableCircumference = 270;
```

Comment:
Why not available _degrees_?

Reply:
Why is this duplicated smh

Reply:
Yea, I agree that might be a better name, git push -f incoming